.env.example

```
@@ -232,3 +234,28 @@ SECRETS_CAP_ENFORCEMENT=
+# ---------------------------------------------------------------------------
+
+# Host interface the app's port is published on. Loopback keeps the plaintext
+# listener local; Caddy (the "tls" profile) reaches the app over the compose
+# network. Use 0.0.0.0 only where a firewall or security group fronts the
+# host, such as behind the AWS load balancer. MinIO's API and console are
+# published on loopback only.
+APP_BIND_ADDRESS=127.0.0.1
+# MinIO root credentials; the AWS_* and LITESTREAM_* credentials above and
+# below are these same values when the stack's MinIO is the object store.
+MINIO_ROOT_USER=minioadmin
+MINIO_ROOT_PASSWORD=minioadmin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ROOT_PASSWORD key should go before the MINIO_ROOT_USER key

(UnorderedKey)